group__enums.html

```diff
@@ -4726,9 +4726,9 @@ <h2 class="memtitle"><span class="permalink"><a href="#gab084d3a6d6ddd144ea6df4a
 </td></tr>
 <tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca1fd9f2d48998b0eb519f4aef9e26784e"></a>DAMAGE_POOP&#160;</td><td class="fielddoc"><p>damage from red poop </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p>damage comes from devil room deal </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p> </p>
```

Comment:
why the removal?

Reply:
Like I mentioned in the PR itself, this is not true. I tested it via debugging and didn't manage to trigger the DAMAGE_DEVIL flag when taking items from the devil deal. If you have more information on this, please share.

Reply:
Ok. When im back from vacation I can take a look at it. So maybe next week.

Reply:
Sounds good.